test/system/160-volumes.bats

```diff
@@ -319,5 +319,44 @@ EOF
     is "$output"  "" "no more volumes to prune"
 }
 
+# Named volumes copyup
+@test "podman volume create copyup" {
+    myvolume=myvol$(random_string)
+    mylabel=$(random_string)
+
+    # Create a named volume
+    run_podman volume create $myvolume
+    is "$output" "$myvolume" "output from volume create"
+
+    # Confirm that it shows up in 'volume ls', and confirm values
+    run_podman volume ls --format json
+    tests="
+Name           | $myvolume
+Driver         | local
+NeedsCopyUp    | true
+"
+    parse_table "$tests" | while read field expect; do
+        actual=$(jq -r ".[0].$field" <<<"$output")
+        is "$actual" "$expect" "volume ls .$field"
+    done
+
+    run_podman run --rm --volume $myvolume:/vol $IMAGE true
+    run_podman volume inspect --format '{{ .NeedsCopyUp }}' $myvolume
+    is "${output}" "true" "NeedsCopyUP should be true"
+
+    run_podman run --rm --volume $myvolume:/etc $IMAGE ls /etc/passwd
+    run_podman volume inspect --format '{{ .NeedsCopyUp }}' $myvolume
+    is "${output}" "false" "NeedsCopyUP should be false"
+
+    run_podman volume inspect --format '{{.Mountpoint}}' $myvolume
+    mountpoint="$output"
+    test -e "$mountpoint/passwd"
+    run_podman volume inspect --format '{{.MountCount}}' $myvolume
+    is "$output" "0" "Mount count should be 0"
```

Comment:
How would you test a nonzero `MountCount`? If it's easy, would you mind adding a test? (I tried real quick but couldn't figure it out)

Reply:
We probably should to this in a separate PR once local mounts are allowed on rootless.  (My other PR).
```
sudo ./bin/podman volume create -o device=tmpfs --opt type=tmpfs dan
$ sudo ./bin/podman volume inspect dan
[
    {
        "Name": "dan",
        "Driver": "local",
        "Mountpoint": "/var/lib/containers/storage/volumes/dan/_data",
        "CreatedAt": "2022-01-04T10:30:45.886433913-05:00",
        "Labels": {},
        "Scope": "local",
        "Options": {
            "device": "tmpfs",
            "type": "tmpfs"
        },
        "MountCount": 0,
        "NeedsCopyUp": true,
        "NeedsChown": true
    }
]
$ sudo ./bin/podman run -d -v dan:/tmp alpine sleep infinity
$ sudo ./bin/podman volume inspect dan
[
    {
        "Name": "dan",
        "Driver": "local",
        "Mountpoint": "/var/lib/containers/storage/volumes/dan/_data",
        "CreatedAt": "2022-01-04T10:30:45.886433913-05:00",
        "Labels": {},
        "Scope": "local",
        "Options": {
            "device": "tmpfs",
            "type": "tmpfs"
        },
        "MountCount": 1,
        "NeedsCopyUp": true
    }
]
 $ sudo podman stop -l -t 0
34cf2f16f45b28ac23a9b66178f82e1dca3c3ebeed6fe3cd6d5d5c17b1f8987b
$ sudo ./bin/podman volume inspect dan
[
    {
        "Name": "dan",
        "Driver": "local",
        "Mountpoint": "/var/lib/containers/storage/volumes/dan/_data",
        "CreatedAt": "2022-01-04T10:30:45.886433913-05:00",
        "Labels": {},
        "Scope": "local",
        "Options": {
            "device": "tmpfs",
            "type": "tmpfs"
        },
        "MountCount": 0,
        "NeedsCopyUp": true
    }
]
```